### PR TITLE
Create RemoveTools.lua

### PR DESCRIPTION
**Pull request by ccuser44**
Removes the tools of the player

### CODE REVIEW

**Review by JEK0x81D — CHANGES_REQUESTED** [review]
Make sure that you're also clearing any tools that may be currently equipped by the player inside the character.

**Comment by ccuser44** [comment]
Thank you for the suggestion 😄

**Review by JEK0x81D — APPROVED** [review]

File: src/ServerScriptService/Server/Packages/RemoveTools.lua [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	Name = "Removetools",
+	Description = "Removes the tools of the player",
+	Location = "Player",
+}
+
+module.Execute = function(Client, Type, Attachment)
+	if Type == "command" then
+		local Player = module.API.getPlayerWithName(Attachment)
+		local BackPack = Player:FindFirstChildOfClass("Backpack")
+		if Backpack then
```
Comment (JEK0x81D) [inline src/ServerScriptService/Server/Packages/RemoveTools.lua:11]: ps Typo here.

Comment (ccuser44) [reply src/ServerScriptService/Server/Packages/RemoveTools.lua:11]: Ok made a pull to fix it.

**Pull request merged**